src/modules/fw_path_navigation/fw_path_navigation_params.c

```diff
@@ -1063,3 +1063,29 @@ PARAM_DEFINE_FLOAT(FW_SPOILERS_LND, 0.f);
  * @group FW Attitude Control
  */
 PARAM_DEFINE_FLOAT(FW_SPOILERS_DESC, 0.f);
+
+/**
+ * Throttle at min airspeed
```

Comment:
```suggestion
 * Required throttle for level flight at minimum airspeed (sea level, standard atmosphere)
 * see: FW_AIRSPD_MIN
```

Reply:
We should keep the first line of the param descriptions as concise as possible, but I'll add your extension to line two.

Reply:
3111a6b362afe19dc1488d5e91ca2a297466be46